resources/public/api/conf/1.0/scenarios/retrieveBusinessDetails.md

```diff
@@ -9,7 +9,44 @@
     <tbody>
         <tr>
             <td><p>N/A - DEFAULT</p></td>
-            <td><p>Simulate a successful response.</p></td>
+            <td><p>Simulate a successful response with a self-employment business.
+            <p>Use the following businessId(s) for this scenario:</p>
```

Comment:
`businessId(s)` -> `businessId` (as Mashuk said brackets are discouraged in GDS)